Dockerfile

```diff
@@ -33,7 +33,9 @@ COPY test/chai-as-promised.js /grist/test/chai-as-promised.js
 COPY app /grist/app
 COPY stubs /grist/stubs
 COPY buildtools /grist/buildtools
+COPY static/locales /grist/static/locales
 RUN yarn run build:prod
+RUN rm -rf /grist/static/locales
```

Comment:
Hmm, I may be wrong, but I fear it would not be as simple as this.

The fact is that `yarn run build:prod` invokes a tool to sanitize the localized strings. If we place them just to make `yarn run build:prod` work, but remove them right after, I think there would be no point for introducing this util: #1354